doc/testplans/material_preview.md

```diff
@@ -0,0 +1,81 @@
+# Material Preview
```

Comment:
I would probably ask someone like @Dan-Linden or @AtlasLinden to review the test plan to make sure it parses well.